person.rb

```diff
@@ -20,6 +21,10 @@ def correct_name
     @name
   end
 
+  def add_rental(rental)
+    rentals << rental
+  end
```

Comment:
- [ ] Instead of creating the rental and then passing it as a parameter here. Kindly pass the parameters so that you can use `self` to achieve the 'has-many' association.

```ruby
  def add_rental(book, date)
    Rental.new(date, book, self)
  end
```